temporian/core/event_set_ops.py

```diff
@@ -1925,6 +1925,52 @@ def select(
 
         return select(self, feature_names=feature_names)
 
+    def select_index_values(
+        self: EventSetOrNode,
+        keys: Optional[Union[IndexKey, List[IndexKey]]] = None,
```

Comment:
What happens when key is None?
(+Do we need it for now?)

Reply:
Had it optional since it will be optional when I add the other possible params this week, but agreed that right now its inconsistent, removed